src/packages/shared-types/opensearch/main/transforms/seatool.ts

```diff
@@ -99,6 +99,21 @@ const getFinalDispositionDate = (status: string, record: SeaTool) => {
     : null;
 };
 
+const isInSecondClock = (data: any) => {
+  if (
+    authorityLookup(data.authority) != "CHIP" && // if it's not a chip
+    [
+      SEATOOL_STATUS.PENDING,
+      SEATOOL_STATUS.PENDING_CONCURRENCE,
+      SEATOOL_STATUS.PENDING_APPROVAL,
+    ].includes(data.seatoolStatus) && // if it's in pending
+    data.raiReceivedDate // if it's latest rai has a received date
+  ) {
+    return true; // then we're in second clock
+  }
+  return false; // otherwise, we're not
+};
```

Comment:
this can just return the condition:

```js
const isInSecondClock = (data: any) => {
  return authorityLookup(data.authority) != "CHIP" && // if it's not a chip
    [
      SEATOOL_STATUS.PENDING,
      SEATOOL_STATUS.PENDING_CONCURRENCE,
      SEATOOL_STATUS.PENDING_APPROVAL,
    ].includes(data.seatoolStatus) && // if it's in pending
    data.raiReceivedDate // if it's latest rai has a received date
}
```